app/_src/using-mesh/managing-ingress-traffic/builtin-listeners.md

```diff
@@ -0,0 +1,360 @@
+---
+title: Configuring built-in listeners
+---
+
+For configuring built-in gateway listeners, use the [`MeshGateway`](/docs/{{ page.version }}/policies/meshgateway) resource.
+
+{% tip %}
+These are {{site.mesh_product_name}} policies so if you are running on multi-zone they need to be created on the Global CP.
+See the [dedicated section](#multi-zone) for using builtin gateways on
+multi-zone.
+{% endtip %}
+
+The `MeshGateway` resource specifies what network ports the gateway should listen on and how network traffic should be accepted.
+A builtin gateway Dataplane can have exactly one `MeshGateway` resource bound to it.
+This binding uses standard, tag-based {{site.mesh_product_name}} matching semantics:
+
+{% tabs binding useUrlFragment=false %}
+{% tab binding Kubernetes %}
+
+```yaml
+apiVersion: kuma.io/v1alpha1
+kind: MeshGateway
+mesh: default
+metadata:
+  name: edge-gateway
+spec:
+  selectors:
+    - match:
+        kuma.io/service: edge-gateway
+```
+
+{% endtab %}
+{% tab binding Universal %}
+
+```yaml
+type: MeshGateway
+mesh: default
+name: edge-gateway
+selectors:
+  - match:
+      kuma.io/service: edge-gateway
+```
+
+{% endtab %}
+{% endtabs %}
```

Comment:
can't we use:

```suggestion
{% policy_yaml binding %}

```yaml
type: MeshGateway
mesh: default
name: edge-gateway
selectors:
  - match:
      kuma.io/service: edge-gateway
```

{% endpolicy_yaml %}
```

?

Reply:
`{% policy_yaml %}` works only with new targetRef policies. In this case it won't create at a fully correct example